e2e/specs/addBiometrics.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { test } from '../core';
+import { BiometricsAndVitalsPage } from '../pages';
+import { expect } from '@playwright/test';
+import { generateRandomPatient, deletePatient, Patient } from '../commands';
+
+let patient: Patient;
+
+test.beforeEach(async ({ api }) => {
+  patient = await generateRandomPatient(api);
+});
+
+test('Record biometrics of the patient', async ({ page, api }) => {
+  const biometricsPage = new BiometricsAndVitalsPage(page);
+
+  await test.step('When I visit the patient biometirics and vitals page', async () => {
+    await biometricsPage.goto(patient.uuid);
+  });
+
+  await test.step('And I start a visit', async () => {
+    await biometricsPage.startVisit();
+    await expect(biometricsPage.page.getByText('Active Visit')).toBeVisible();
```

Comment:
To start a visit I think it's better to use the API. Otherwise all these tests will fail if there's an issue with the starting a visit. WDYT? @RandilaP @Piumal1999 @jayasanka-sack

Reply:
I asked from @jayasanka-sack and he says to use the UI

Reply:
@jayasanka-sack Using the UI will fail all tests if there's a bug in the active visit UI. Ideally, it should not fail all the tests because it's not a part of the test.